rector.php

```diff
@@ -46,7 +46,7 @@
         SetList::PHP_83,
         PHPUnitSetList::PHPUNIT_90,
         PHPUnitSetList::PHPUNIT_CODE_QUALITY,
-        SymfonySetList::SYMFONY_71,
+        SymfonySetList::SYMFONY_72,
```

Comment:
Need https://github.com/rectorphp/rector-symfony/pull/684

Reply:
Lets do this in a follow up PR?

Reply:
This is still open, right?

Reply:
Rector 2.0 has been released, but it need to update phpstan 2.x Aldo.

And there is a conflict